doc/source/ray-core/examples/batch_prediction.ipynb

```diff
@@ -7,7 +7,7 @@
     "# Batch Prediction\n",
     "\n",
     "```{tip}\n",
-    "Ray provides multiple ways to implement this workload.\n",
+    "Ray provides multiple ways to implement this workload. As a general rule, for simple, naively parallel workloads like this one:\n",
     "\n",
     "* Ray Core will be the lowest level of abstraction and the most performant.\n",
     "* Ray Tune will provide fault tolerance, monitoring and integration with experiment management, but will be slower than Ray Core.\n",
```

Comment:
IMO, if we ever want to recommend Core, the more likely reason would be flexible rather than performance.

Reply:
ATM the performance difference between Core and libraries is very significant. While we should obviously work on reducing that gap, many users have had better results with Core for that reason